open-craft/opencraft
# Remove ephemeral databases support

This pull request removes support for ephemeral databases.

**Testing instructions**:

1. Spin up an OCIM devstack
2. Create an instance, checking that the `use_ephemeral_databases` option disappeared.
3. Spin up an AppServer for given instance.
4. Check that the AppServer makes use of external databases.

**Author notes and concerns**:

~1. Some unit tests are failing as the provided .env.test still does not have external servers configured. This will be fixed asap.~

**Reviewers**

**Changes included**
Modify models to remove field
Provides migration
Fix templates and serialization
Remove stale tests for ephemeral support
Fix remaining tests
Adjust README for given changes
